src/main/java/org/chzz/market/domain/auctionv2/repository/AuctionV2QueryRepository.java

```diff
@@ -122,6 +146,76 @@ public Optional<OfficialAuctionDetailResponse> findOfficialAuctionDetailById(Lon
         return officialAuctionDetailResponse;
     }
 
+    /**
+     * 사전 경매 목록 조회
+     */
+    public Page<PreAuctionResponse> findPreAuctions(Long userId, Category category, Pageable pageable) {
+        List<PreAuctionResponse> content = jpaQueryFactory.from(auctionV2)
+                .select(
+                        Projections.constructor(
+                                PreAuctionResponse.class,
+                                auctionV2.id,
+                                auctionV2.name,
+                                imageV2.cdnPath,
+                                auctionV2.minPrice.longValue(),
+                                userIdEq(userId),
+                                auctionV2.likeCount,
+                                likeV2.id.isNotNull()
+                        )
+                )
+                .from(auctionV2)
+                .join(auctionV2.seller, user)
+                .leftJoin(imageV2).on(imageV2.auction.eq(auctionV2).and(isRepresentativeImage()))
```

Comment:
`imageV2`가 `auctionV2`와 N:1관계로 기억는데 앞의 조건절은 명시적으로 연관관계임을 나타내기 위함인가요?

N:1관계라 아래처럼 해도 될거같아서요
```suggestion
                .leftJoin(imageV2).on(isRepresentativeImage())
```

Reply:
연관관계 이용해서 수정했습니다
```
    left join
        imagev2 i1_0 
            on av1_0.auction_id=i1_0.auction_id 
            and i1_0.sequence=1 
```